### PR TITLE
docs: Fix all credential documentation urls, and add a CI job to regularly validate these urls

### PR DESCRIPTION
**Pull request by netroy**

### CODE REVIEW

File: scripts/validate-docs-links.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		if (statusCode !== 200) invalidUrls.push(name);
+	}
+
+	if (missingDocs.length) console.log('Documentation URL missing for %s', kind, missingDocs);
+	if (invalidUrls.length) console.log('Documentation URL invalid for %s', kind, invalidUrls);
+	if (missingDocs.length || invalidUrls.length) process.exit(1);
+};
+
+(async () => {
+	await checkLinks('credentials');
```
Comment (netroy) [inline scripts/validate-docs-links.js:52]: I'll add the check for nodes in another PR.

File: packages/nodes-base/credentials/FacebookGraphApi.credentials.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export class FacebookGraphApi implements ICredentialType {
 
 	displayName = 'Facebook Graph API';
 
-	documentationUrl = 'facebookGraph';
+	documentationUrl = 'facebookgraph';
```
Comment (netroy) [inline packages/nodes-base/credentials/FacebookGraphApi.credentials.ts:13]: I'm checking why this was not caught by the script

File: scripts/validate-docs-links.js (hunk 1 above)
Comment (janober) [reply scripts/validate-docs-links.js:62]: :+1:

File: packages/nodes-base/credentials/FacebookGraphApi.credentials.ts (hunk 2 above)
Comment (janober) [reply packages/nodes-base/credentials/FacebookGraphApi.credentials.ts:13]: :pray:

**Review by janober — DISMISSED** [review]

**Review by janober — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.209.1`

**Pull request merged**